pa11y_scripts/.pa11y-local.json

```diff
@@ -21,12 +21,28 @@
           "actions": [
             "wait for element #id_0-contact_phone to be visible",
             "click element #submit-next",
-            "wait for element #id_1-protected_class to be visible",
-            "click element #id_1-protected_class_1",
-            "click element #submit-next"
+            "wait for element #id_1-primary_complaint to be visible",
+            "click element #id_1-primary_complaint_0",
+            "click element #submit-next",
+            "wait for element #id_2-protected_class to be visible"
           ],
           "screenCapture": "pa11y-screenshots/page3.png"
         },
+        {
+          "url": "http://0.0.0.0:8000/report",
+          "actions": [
+            "wait for element #id_0-contact_phone to be visible",
+            "click element #submit-next",
+            "wait for element #id_1-primary_complaint to be visible",
+            "click element #id_1-primary_complaint_0",
+            "click element #submit-next",
+            "wait for element #id_2-protected_class to be visible",
+            "click element #id_2-protected_class_0",
+            "click element #submit-next",
+            "wait for element #3-violation_summary to be visible"
```

Comment:
I figured out my issue, for this line I needed it to be `#id_3-violation_summary` for it to run.

Reply:
Oooooh, great catch! Thank you @LindsayYoung!